### DISCUSSION TITLE
post is visible localhost:4000, but not visible https://~.github.io

### DISCUSSION THREAD

**Discussion by bom04** [mmistakes/minimal-mistakes#3465] [Q&A] [answered]
### What happened?

- Minimal Mistakes version: latest version
- Ruby gem or remote theme version: ruby 3.0.3p157 (2021-11-24 revision 3fb7d2cadc) [x64-mingw32]
- Jekyll version: jekyll 4.2.1
- Git repository URL: https://github.com/bom04/bom04.github.io
- Hosted on GitHub Pages (if yes provide URL to site): https://bom04.github.io/
- Operating system: windows 10

### Expected behavior

All posts are visible on the localhost:4000, but not on https://.../github.io.
When I run local server, I see below message, but it works well..Could this message be the cause of the problem?
```
 Conflict: The following destination is shared by multiple files.
                    The written file may end up with unexpected contents.
                    C:/bom04.github.io/_site/index.html
                     - _pages/home.md
                     - index.html
```

Or is there some other problem?

### Steps to reproduce the behavior

I proceeded with the minimal-mistakes download zip not fork.
So, when I first pushed the minimal-mistakes on my GitHub, the post I wrote in advance was also pushed. At this time, the registered posts are displayed not only locally but also on the website.
After that, I did not change any settings and only pushed posts. However, posts registered after this are only visible locally and not on the website.

### Other

I already have index.html in root
```
---
layout: home
author_profile: true
--- 
```
and already set paginate, paginate_path and jekyll-paginate plugins in _config.yml
```
paginate: 5 # amount of posts to show
paginate_path: /page:num/

# Plugins (previously gems:)
plugins:
  - jekyll-paginate
```

please help me...

**Comment by mmistakes** [comment, marked as answer]
Jekyll is telling you what the issue is with:

```
Conflict: The following destination is shared by multiple files.
                    The written file may end up with unexpected contents.
                    C:/bom04.github.io/_site/index.html
                     - _pages/home.md
                     - index.html
```

It's finding a conflict between two files that are both trying to write to the same location `index.html`.
If you look at `_pages/home.md` you'll see that it's setting a `permalink` that is the same as the `index.html` file in the root of your site.

The fix is to remove `_pages/home.md` so Jekyll stops trying to override your paginated home `index.html` file. My guess is you forked the repo and didn't clear out all of the demo content.
